src/common/vulkan_platform.h

```diff
@@ -32,9 +32,19 @@
 // redefined to be nullptr). This keeps the type-safety of having the handles be different types
 // (like vulkan.h on 64 bit) but makes sure the types are different on 32 bit architectures.
 
+// Force the handle type to have the same alignment as what would have been the Vulkan
+// non-dispatchable handle type.
+#if DAWN_PLATFORM_64_BIT
+#    define ALIGNAS_VULKAN_HANDLE alignas(alignof(struct VkSomeHandle*))
```

Comment:
Kind of unclear where the definition of VkSomeHandle is coming from (after a minute I realized this IS the definition) - a comment could be nice. (Or maybe use a real struct that's already in scope?)

Reply:
Added a comment, also changed code in the #if to declare a type. This way the alignas(alignof()) is not duplicated.